python/cudf/cudf/dataframe/datetime.py

```diff
@@ -235,6 +236,28 @@ def find_last_value(self, value):
         value = columnops.as_column(value).as_numerical[0]
         return self.as_numerical.find_last_value(value)
 
+    def unique(self, method="sort"):
+        # method variable will indicate what algorithm to use to
+        # calculate unique, not used right now
+        if method != "sort":
+            msg = "non sort based unique() not implemented yet"
+            raise NotImplementedError(msg)
+        segs, sortedvals = self._unique_segments()
+        # gather result
+        out_col = cpp_copying.apply_gather_array(sortedvals, segs)
+        return out_col
```

Comment:
Any reason this couldn't just be moved to the column class?

Reply:
I was going to move `unique` as well, but noticed that the implementation is not consistent across the sub-classes (categorical is different). Rather than have Numerical and Datetime inherit their `unique` implementation from Column but Categorical override, I left it as is. It felt a bit weird to override a fully-fleshed out method from the parent class compared to overriding a placeholder method.

Happy to move it if you think it's worth it, though @kkraus14 .

Reply:
👍 lets just leave it as is for now then, can refactor later.